File: src/main/java/top/zbeboy/isy/web/bean/internship/statistics/InternshipChangeCompanyHistoryBean.java
package top.zbeboy.isy.web.bean.internship.statistics;

import lombok.Data;
import lombok.EqualsAndHashCode;
import top.zbeboy.isy.domain.tables.pojos.InternshipChangeCompanyHistory;

/**
 * Created by lenovo on 2016-12-12.
 */
@EqualsAndHashCode(callSuper = false)
@Data
public class InternshipChangeCompanyHistoryBean extends InternshipChangeCompanyHistory {
    private String internshipTitle;
    private String realName;
    private String studentNumber;
    private String organizeName;
    private String changeTimeStr;
}

File: src/main/java/top/zbeboy/isy/web/bean/internship/statistics/InternshipChangeCompanyHistoryBean.kt
package top.zbeboy.isy.web.bean.internship.statistics

import top.zbeboy.isy.domain.tables.pojos.InternshipChangeCompanyHistory

/**
 * Created by zbeboy 2017-12-28 .
 **/
class InternshipChangeCompanyHistoryBean : InternshipChangeCompanyHistory() {
    var internshipTitle: String? = null
    var realName: String? = null
    var studentNumber: String? = null
    var organizeName: String? = null
    var changeTimeStr: String? = null
}